File: fix_cargo.py
with open('src-tauri/Cargo.toml', 'r') as f:
    c = f.read()

if 'tauri-plugin-shell' not in c:
    c = c.replace('tauri-plugin-fs = "2"', 'tauri-plugin-fs = "2"\ntauri-plugin-shell = "2"')
    with open('src-tauri/Cargo.toml', 'w') as f:
        f.write(c)
    print('Added tauri-plugin-shell')
else:
    print('Already present')

File: fix-cargo.js
const fs = require('fs');
let content = fs.readFileSync('src-tauri/Cargo.toml', 'utf8');
content = content.replace(/crates\/(audio|stt|bible|detection|broadcast|api|notes)/g, 'crates/lifestone-');
fs.writeFileSync('src-tauri/Cargo.toml', content);
console.log('Fixed Cargo.toml');
